packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -15,12 +15,10 @@ import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
 
 import 'editable_text_utils.dart';
 
-final TextEditingController controller = TextEditingController();
```

Comment:
Instead of completely removing these variables from shared scope, can their construction and disposal be moved to global setUp and tearDown for this lib?

Reply:
Nice catch! I updated this test. Could you please check?